tests/vector/test_spaces.py

```diff
@@ -129,3 +133,79 @@ def test_iterate_custom_space(space, batch_space):
     for i, item in enumerate(iterator):
         assert item in space
     assert i == 3
+
+
+seeded_spaces = [
```

Comment:
I suggest you reuse the `spaces` and `expected_batched_spaces_4` and so-on fixtures from above, and just seed them.

Reply:
I removed `seeded_spaces` in 296cf50def0ef62b3d47f56f2cd98628806e36e9